actions/parsing.py

```diff
@@ -55,7 +58,8 @@ def parse_duckling_time_as_interval(
         return make_interval_from_value_duckling_time(timeinfo)
 
 
-def format_time_by_grain(time, grain=None):
+def format_isotime_by_grain(isotime, grain=None):
+    time = parser.isoparse(isotime)
```

Comment:
`time` is a standard python library, so suggesting a rename here

```suggestion
    value = parser.isoparse(isotime)
```

Reply:
ooh good point